ui/client/components/PeriodicallyReloadingComponent.js

```diff
@@ -1,14 +1,19 @@
-import React from "react";
-
+import React from "react"
 
 class PeriodicallyReloadingComponent extends React.Component {
+  baseIntervalTime = 10000
```

Comment:
we have two defaults now - here and in backend? Let's leave only one?

Reply:
I'v changed it. Now default is only at frontend.